packages/fields/src/types/File/README.md

```diff
@@ -33,7 +33,7 @@ keystone.createList('Applicant', {
 
 | Option       | Type      | Default  | Description                                                                                            |
 | ------------ | --------- | -------- | ------------------------------------------------------------------------------------------------------ |
-| `adapter`    | `Object`  | Required | See the [File Adapters](https://www.keystonejs.com/keystone/file-adapters/) page for more information. |
+| `adapter`    | `Object`  | Required | See the [File Adapters](/keystonejs/file-adapters/) page for more information. |
```

Comment:
```suggestion
| `adapter`    | `Object`  | Required | See the [File Adapters](https://www.keystonejs.com/keystonejs/file-adapters/) page for more information. |
```

Reply:
We use absolute URLs in README.MD files so they work on github.

Reply:
Ok, I found both absolute and relative URLs, so I opted for relative... check e.g. file packages/fields/README.md, there are both absolute and relative URLs in there.

Reply:
Good catch, those should be absolute as well.